cohortextractor/patients.py

```diff
@@ -1923,5 +1923,19 @@ def with_high_cost_drugs(
     return "with_high_cost_drugs", locals()
 
 
-def ethnicity_from_sus(*, setting, on_conflict):
-    pass
+def with_ethnicity_from_sus(returning=None):
+    """
+    Returns ethnicity data from the SUS Datasets
+
+    Args:
```

Comment:
I think this needs an extra keyword arg to describe the matching logic. Something like:
```
  use_most_recent_code=None
```
This wouldn't _do_ anything other than we'd have an `assert use_most_recent_code` somewhere to make sure that it was being passed as an argument. This means that's it's hopefully obvious to anyone reading the study def what the logic is. And it future-proofs us for when we add `use_most_frequent_code`.

Reply:
Nice.  I think you got the names the wrong way round?  But if anything that just shows why it's good 😁

Added, documented, and rebased in.

Reply:
Ah, yes! Exactly!